Not working with React Native 0.60

I've created a new React Native app with the latest version (0.60), and installed this module with the steps in the README, but since it introduced autolinking (https://github.com/react-native-community/cli/blob/master/docs/autolinking.md), when compiling, it just gives some errors, even when I was linking it manually (aka the way described in the README). 

```
/home/ben/code/spoti-find/test/android/app/build/generated/rncli/src/main/java/com/facebook/react/PackageList.java:19: error: cannot find symbol
import com.spotify.sdk.android.authentication.RNSpotifyPackage;
                                             ^
  symbol:   class RNSpotifyPackage
  location: package com.spotify.sdk.android.authentication
/home/ben/code/spoti-find/test/android/app/build/generated/rncli/src/main/java/com/facebook/react/PackageList.java:54: error: cannot find symbol
      new RNSpotifyPackage()
          ^
  symbol:   class RNSpotifyPackage
  location: class PackageList
2 errors
```

I know this error is related to the import of the wrong code, since it still tries to autolink it if I manually linked it.

It may also have to do with AndroidX support, since that was also introduced. The jetifier could be used, if its related to AndroidX, https://github.com/mikehardy/jetifier.

By making this module compatible with React Native 0.60 #135, #138, and #137 would be fixed with the autolinking (I think).

Or if anyone is able to make it work with React Native 0.60, please let me know :)

--- a/react-native.config.js
+++ b/react-native.config.js
@@ -1,0 +1,9 @@
+module.exports = {
+  dependency: {
+    platforms: {
+      android: {
+        packageImportPath: 'com.lufinkey.react.spotify.RNSpotifyPackage',
+      },
+    },
+  },
+};

